src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars__star--active {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
here you should write a solution using modifier for `stars`.
For stars--1 you will have one formula. For stars--2 you'll have another formula and so on. 

Example:
```
.stars--1 .stars__star:nth-child(_your_formula_) {
  background-image: url(images/star-active.svg);
}
```